en/docs/try-out/event-notification-api.md

```diff
@@ -1,4 +1,9 @@
-This document provides step by step instructions to deploy, subscribe, and invoke the Event Notifications Aggregated Polling API.
+There are two APIs under the Event Notification API:
```

Comment:
Two apis we support

Reply:
Fixed in https://github.com/wso2/docs-ob-toolkit-uk/pull/80/commits/c7d79fa574f545432878e1c445eb6b44781916b8.